script/get-oneapi/meta.yaml

```diff
@@ -35,12 +35,12 @@ tests:
       - fortran
 
 versions:
-  2025.1.1:
+  2025.2.0:
```

Comment:
Thank you for your PR. Can you please add `2025.2.0` as an addition and not remove `2025.1.1`?